airbyte-integrations/bases/base-normalization/unit_tests/test_destination_name_transformer.py

```diff
@@ -117,7 +117,7 @@ def test_transform_standard_naming(input_str: str, expected: str):
         ("a-Unicode_name_文6", "MSSQL", "a_unicode_name__6", "{{ adapter.quote('a-Unicode_name_文6') }}"),
         # Doesnt start with alpha or underscore
         ("100x2001", "Postgres", "100x2001", "{{ adapter.quote('100x2001') }}"),
-        ("100x2002", "BigQuery", "_100x2002", "_100x2002"),
```

Comment:
Should the input of "_100x2002" be transformed into "100x2002" for BIGQUERY specifically, based on this use case?